.github/workflows/danger-workflow.yml

```diff
@@ -0,0 +1,24 @@
+name: Danger
+on: 
+  pull_request:
+    # By default, the pull_request event only triggers on [opened, synchronize, reopened]
+    types: [opened, synchronize, reoponed, assigned, unassigned, labeled, unlabeled]

```

Comment:
So I added the label, the Danger/Danger CI check ran for a third time, and the comment disappeared. Looks like it works. And I agree that these are the events we need to re-run on, since the other events listed in the reference would not affect the pass/fail state of Danger